src/shared/markdown-serializer.ts

```diff
@@ -597,7 +597,12 @@ const extendedCodeMarkDeserializer: typeof defaultCodeMarkDeserializer = {
 // extend the default markdown serializer's marks and add our own
 const customMarkdownSerializerMarks: MarkdownSerializerMarks = {
     ...defaultMarkdownSerializer.marks,
-    em: genMarkupAwareMarkConfig(defaultMarkdownSerializer.marks.em),
+    em: genMarkupAwareMarkConfig({
```

Comment:
If we change the default here, we also need to change it in the commonmark editor. Otherwise, we'll just end up in the same boat.

Reply:
I assume that's handled by this change: https://github.com/StackExchange/Stacks-Editor/pull/161/files#diff-98cdb9c85b4c07d8f39aa6d21d0934ca06651821fbbf3a774d55b09ce04df62dR687